package.json

```diff
@@ -20,9 +39,9 @@
     "storybook": "run-s build:icons:index storybook:dev",
     "storybook:dev": "start-storybook --port 9001 --config-dir .storybook --static-dir ./public",
     "build-storybook": "build-storybook --config-dir .storybook --output-dir .out --static-dir ./public",
-    "clean": "rimraf dist lib src/icons/**/svgs",
-    "build": "run-s clean && run-s build:icons:all && run-s tsc && run-p build:rollup build:babel && run-p bundlewatch",
```

Comment:
It does not look like the rollup build fails when there's type errors:

```
> radiance-ui@22.0.0 build:rollup
> rollup -c


src/index.ts, src/constants/index.ts, src/icons/index.ts, src/utils/index.ts → dist, dist...
(!) Plugin typescript: @rollup/plugin-typescript TS2322: Type 'Validator<string>' is not assignable to type 'Validator<boolean>'.
  Type 'string' is not assignable to type 'boolean'.
src/shared-components/chip/index.tsx: (92:3)

92   text: PropTypes.string.isRequired,
     ~~~~

(!) Plugin typescript: @rollup/plugin-typescript TS2322: Type 'string' is not assignable to type 'boolean'.
src/shared-components/chip/test.tsx: (9:59)

9       const { container } = render(<Chip status="success" text="Success" />);
                                                            ~~~~

  src/shared-components/chip/index.tsx:13:3
    13   text: boolean;
         ~~~~
    The expected type comes from property 'text' which is declared here on type 'IntrinsicAttributes & ChipProps & { children?: ReactNode; }'

created dist, dist in 24.4s

[...etc]
```

I think we should add back the `tsc` step. 

(Related to your other comment, if we want to DRY up the steps then the first step of `build` can be `build:icons:all` to bake in the clean step, and then it's just build icons + tsc + rollup + bundlewatch).

Reply:
Or this option is probably better: https://github.com/rollup/plugins/tree/master/packages/typescript#noemitonerror

Reply:
Oh good catch, didn't realize that wasn't the default. 👍 to the build:icons:all idea too.

Reply:
Confirmed the build fails on type errors with the changes in ef585a0